style.css

```diff
@@ -0,0 +1,46 @@
+#nav-bar {
+  overflow: hidden;
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+}
+
+body {
+  background-color: lightblue;
+}
+
+@media screen and (min-width: 400px) {
+  body {
+    background-color: lightgreen;
+  }
+
+  #header-img {
+    width: 250px;
+  }
+}
+
+@media screen and (min-width: 800px) {
+  body {
+    background-color: lavender;
+  }
+
+  #header-img {
+    width: 500px;
+  }
+}
+
+.flex-container {
+  display: flex;
+  flex-direction: row;
+}
+
+.flex-container > div {
+  background-color: dodgerblue;
+  color: white;
+  width: 100px;
+  margin: 10px;
+  text-align: center;
+  line-height: 75px;
+  font-size: 30px;
+}
```

Comment:
The design elements used inside flex-container do the job but are not properly aligned with the rest of the elements on-page. They overlap with the rest of the elements and become unclear. Can you kindly recheck the alignment of the objects inside flex-container and adjust accordingly?